src/commands/script-dump.json

```diff
@@ -26,8 +26,8 @@
                     "type": "string"
                 },
                 {
-                    "description": "null if sha1 hash does not exist in script cache.",
-                    "type": "null"
+                    "description": "noscript error if sha1 hash does not exist in script cache.",
+                    "const": "NOSCRIPT No matching script."
```

Comment:
This is not how we normally describe an error reply.

I think we just don't describe error replies in the reply schema. They're just mentioned in the docs if I remember correctly.

Reply:
Got it